sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,65 @@
+# 2021 SIG network annual report
+
+This report reflects back on CY 2020 and was written in April 2021.
+
+> Note: Each response should have supporting documentation, linked KEPs, graphs, data
+> Note: Err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+# Operational
+
+- How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+  - Is your [README](./README.md) accurate? 
+    - Yes, our README is up to date.
+  - Have a CONTRIBUTING.md file?
+    - No - we do not have a CONTRIBUTING.md file.
+  - All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sig-list.md)?
+    - Yes, all subprojects are correctly listed. 
+    - We have one unofficial group that should probably be a subproject - the network plumbing group.
```

Comment:
Can you provide a link to any materials?
Are meetings open and available for all to attend?
Can the SIG take an action to clarify its future?

Reply:
Are meetings recorded and uploaded similar to normal SIG meetings?

Reply:
@dcbw @dougbtv @squeed -- Do you have an answers to Derek's questions?

The agenda and minutes are at: https://docs.google.com/document/d/1oE93V3SgOGWJ4O1zeD1UmpeToa0ZiiO6LqRAmZBPFWM/edit and that has the meeting details.